cliUsage.html

```diff
@@ -135,6 +138,9 @@ <h5>If you want to have the message publishing on RabbitMQ with given routing ke
         <h5>If you want to have the message publishing on RabbitMQ with given tag:</h5>
         <pre><code>java -jar publish-cli.jar -f publishMessages.json -en rabbit -mb mb123 -mp eiffelsemantics -tag production</code></pre>
 
+        <h5>If you want to have the message publishing on RabbitMQ by creating an exchange with given create_exchange:</h5>
```

Comment:
If there are no exchanges, then we can create an exchange and publish the messages to it by using create_exchange property. Example: